ktlint-ruleset-standard/src/test/kotlin/com/pinterest/ktlint/ruleset/standard/rules/PropertyNamingRuleTest.kt

```diff
@@ -158,4 +159,50 @@ class PropertyNamingRuleTest {
             """.trimIndent()
         propertyNamingRuleAssertThat(code).hasNoLintViolations()
     }
+
+    @Nested
+    inner class `Given property is serialVersionUID` {
+        @Test
+        fun `Given property is present in companion object`() {
+            val code =
+                """
+                class Foo1 {
+                    companion object {
+                        private const val serialVersionUID: Long = 123
+                    }
+                }
+                class Foo2 {
+                    companion object {
+                        private val serialVersionUID: Long = 123
+                    }
+                }
+                class Foo3 {
+                    companion object {
+                        @JvmStatic private val serialVersionUID: Long = 123
+                    }
+                }
+                class Foo4 {
+                    private val serialVersionUID: Long = 123
+                }
```

Comment:
What is the value of those additional test cases? You only want to verify that a const property with identifier `serialVersionUID` is not being reported as violation.

Reply:
Hi @paul-dingemans this TC is just added to ensure that normal `class` remains same with future iteration of this rule.

Reply:
Those cases were already *not* reported before because the `const` modifier was missing. Adding an extra exclusion based on name will never interfere with that.

Reply:
Got it. Let me remove these TCs

Reply:
Updated